nebari/initialize.py

```diff
@@ -331,16 +331,16 @@ def render_config(
 
     # Save default password to file
     default_password_filename = os.path.join(
-        tempfile.gettempdir(), "QHUB_DEFAULT_PASSWORD"
+        tempfile.gettempdir(), "NEBARI_DEFAULT_PASSWORD"
     )
     with open(default_password_filename, "w") as f:
         f.write(default_password)
     os.chmod(default_password_filename, 0o700)
 
-    config["theme"]["jupyterhub"]["hub_title"] = f"QHub - { project_name }"
+    config["theme"]["jupyterhub"]["hub_title"] = f"Nebari - { project_name }"
     config["theme"]["jupyterhub"][
         "welcome"
-    ] = f"""Welcome to { qhub_domain }. It is maintained by <a href="http://quansight.com">Quansight staff</a>. The hub's configuration is stored in a github repository based on <a href="https://github.com/Quansight/qhub/">https://github.com/Quansight/qhub/</a>. To provide feedback and report any technical problems, please use the <a href="https://github.com/Quansight/qhub/issues">github issue tracker</a>."""
+    ] = f"""Welcome to { nebari_domain }. It is maintained by <a href="http://quansight.com">Quansight staff</a>. The hub's configuration is stored in a github repository based on <a href="https://github.com/Quansight/nebari/">https://github.com/Quansight/nebari/</a>. To provide feedback and report any technical problems, please use the <a href="https://github.com/Quansight/nebari/issues">github issue tracker</a>."""
```

Comment:
Changes needed:
- maintained -> nebari dev team (use nebari.dev url)
- fix URLs
- GitHub capitalization

Reply:
copied from previous review message:
there is a PR for updates to the theme and welcome message https://github.com/Quansight/qhub/pull/1503. Personally, I think we should leave it to that PR to make these changes.